python/src/aiconfig/editor/server/server.py

```diff
@@ -829,3 +829,25 @@ def set_aiconfigrc() -> FlaskResponse:
     # yaml = YAML()
     # with open(state.aiconfigrc_path, "w") as f:
     #     yaml.dump(request_json["aiconfigrc"], f)
+
+@app.route("/api/set_env_file_path", methods=["POST"])
+def set_env_path() -> FlaskResponse:
+    """
+    Updates the server state with the new .env file path.
+
+    This method does NOT load the .env file into the server state.
+    It only updates the path to the .env file. See api/run
+
+    """
+    request_json = request.get_json()
+    state = get_server_state(app)
+
+    # Validate
+    request_env_path = request_json.get("env_file_path")
```

Comment:
Can we do basic validation of the path here as well? It would seem really weird to successfully set an invalid path and only learn that it's not valid when I go to run a prompt later

Reply:
yea we could but vscode already validates that. Can add anyway for more robustness if you think we should add it

Reply:
Just for the sake of correctness, I think we should. Maybe just:
- path exists / is accessible
- filename format is correct

Reply:
basic validation sounds good to me, added and updated with testplan